content/blog/2018/01/2018-01-15-overhaul-of-manage-jenkins-page.adoc

```diff
@@ -0,0 +1,142 @@
+---
+layout: post
+title: "Overhaul of Manage Jenkins page"
+tags:
+- jenkins
+- ui
+- restyling
+author: recena
```

Comment:
FYI author page is missing in https://github.com/jenkins-infra/jenkins.io/tree/master/content/_data/authors . I would also suggest adding an avatar to https://github.com/jenkins-infra/jenkins.io/tree/master/content/images/avatars (new feature)

Reply:
I'll address this subject in a new PR.